Sources/WalletConnect/KeyManagementServiceProtocol.swift

```diff
@@ -2,20 +2,5 @@
 import Foundation
```

Comment:
This file is now empty, can we just delete it?

Reply:
done